.changelog/15522.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+server: Added raft snapshot arguments to server config
+```
```

Comment:
I can't make the "suggestion" work here in the code review interface because of the triple backticks, but maybe we want to add a second improvement for this same entry:

> server: Raft configuration can now be reloaded without restarting the server